cluster/charts/rook-ceph/templates/resources.yaml

```diff
@@ -55,12 +62,122 @@ spec:
                   type: boolean
             storage:
               properties:
+                useAllNodes:
+                  type: boolean
                 nodes:
-                  items: {}
+                  items:
+                    properties:
+                      name:
+                        type: string
+                      config:
+                        properties:
+                          metadataDevice:
+                            type: string
+                          storeType:
+                            type: string
+                            pattern: ^(filestore|bluestore)$
+                          databaseSizeMB:
+                            type: string
+                          walSizeMB:
+                            type: string
+                          journalSizeMB:
+                            type: string
+                          osdsPerDevice:
+                            type: string
+                          encryptedDevice:
+                            type: string
+                            pattern: ^(true|false)$
+                      useAllDevices: 
+                        type: boolean
+                      deviceFilter: {}
+                        # Needs K8s >= 1.14
+                        # type: string
+                        # nullable: true
+                      directories:
+                        type: array
+                        items:
+                          properties:
+                            path:
+                              type: string
+                      devices:
+                        type: array
+                        items:
+                          properties:
+                            name:
+                              type: string
+                            config: {}
+                              # Needs K8s >= 1.14
```

Comment:
Instead of keeping all these comments inline, how about if we open an issue to track it? Comments like this tend to go stale pretty quickly. It will be a while before 1.14 is our minimum version, so we will need to re-evaluate later anyway.